### DISCUSSION TITLE
Calculator App

### DISCUSSION THREAD

**Discussion by SarthakTools** [TomSchimansky/CustomTkinter#2602] [Showcase]
Hello @TomSchimansky,

I believe in empowering others with accessible tools that simplify everyday tasks. To achieve this, I’ve created a user-friendly calculator app using Python and Tkinter, designed to enhance productivity. 
Here’s the link to the app: [Calculator App on GitHub](https://github.com/SarthakTools/Calculator).

**Comment by MMETehrani** [comment]
Great initiative! A user-friendly calculator is always handy. Looking forward to checking it out!

**Reply by SarthakTools** [reply]
Yeah thanks for replying.
